npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker.rb

```diff
@@ -183,7 +186,7 @@ def latest_version_resolvable_with_full_unlock?
       end
 
       def updated_dependencies_after_full_unlock
-        return conflicting_updated_dependencies if !dependency.top_level? && security_advisories.any?
+        return conflicting_updated_dependencies if security_advisories.any? && vulnerability_audit["fix_available"]
```

Comment:
please can we get some test coverage for this too? Feel free to skip if I missed this already :)